server/lib/src/server/migrations.rs

```diff
@@ -895,6 +897,23 @@ impl<'a> QueryServerWriteTransaction<'a> {
                 err
             })?;
 
+        // all the built-in objects get a builtin class
+        let filter = f_and(vec![
+            f_lt(
+                Attribute::Uuid,
+                PartialValue::Uuid(uuid::uuid!("00000000-0000-0000-0001-000000000000")),
+            ),
+            f_or(vec![
```

Comment:
Why the or? Just tag everything that ships as built in, including schema because then we can tell what's ours and what's not.

Reply:
Because it was breaking all the replication tests before I added this, complaining that "builtin" wasn't valid.

Reply:
Can you mak ethe change on the PR then I'll solve the replication issue?

Reply:
so just drop the and/or combo on filter on lt(uuid)?

Reply:
Yep